src/test/java/com/flightstats/hub/cluster/DistributedLockRunnerTest.java

```diff
@@ -71,10 +70,10 @@ public void test_runWithLock_withTwoLockables_runsOneAtATime() throws Exception
         });
 
         executorService.execute(() -> {
-            Consumer<LeadershipLock> lockable = leadershipLock -> {
-                doAThing(0, "lockable2", latch);
-            };
-            try { waitForMe.await(90, TimeUnit.MILLISECONDS); } catch (Exception e) { }
+            Consumer<LeadershipLock> lockable = leadershipLock -> doAThing(0, "lockable2", latch);
+            try { waitForMe.await(90, TimeUnit.MILLISECONDS); } catch (Exception e) {
```

Comment:
Does it make sense to fail the test if it gets here? I know this is unrelated to the work you were doing...

Reply:
I'll take a look